pkgs/development/python-modules/pandoc-plantuml-filter/default.nix

```diff
@@ -0,0 +1,26 @@
+{ buildPythonPackage
+, fetchPypi
+, pandocfilters
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "pandoc-plantuml-filter";
+  version = "0.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "08673mfwxsw6s52mgglbdz7ybb68svqyr3s9w97d7rifbwvvc9ia";
+  };
+
+  propagatedBuildInputs = [
+    pandocfilters
+  ];
+
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

Reply:
There are no tests.

How do I use pythonImportsCheck?

Reply:
import <pkg-name> -> `pythonImportsCheck = [ "<pkg-name>" ];

Reply:
There are no modules to import. It only exposes a python script that is executed.

Reply:
https://github.com/timofurrer/pandoc-plantuml-filter